proto/interchain_security/ccv/consumer/v1/genesis.proto

```diff
@@ -29,15 +41,6 @@ message GenesisState {
   string provider_channel_id = 3;
   // true for new chain, false for chain restart.
   bool new_chain = 4;
-  // !!! DEPRECATED !!! ProviderClientState is deprecated. Use provider.client_state instead
```

Comment:
Removing this requires that we have yet another consumer genesis migration CLI command.
Could we avoid this change and simply mark it as no-op?

Or is it so that the provider hasn't even returned these fields for a while now?

Reply:
The provider doesn't return these fields for a while.

- `provider_client_state`, `provider_consensus_state` and `initial_val_set` are part of the ProviderInfo
- `maturing_packets` is not return by the provider as it's empty for new consumer modules